coordinator/src/node.rs

```diff
@@ -207,25 +208,31 @@ impl Node {
 
         let channel_details = self.get_counterparty_channel(trade_params.pubkey)?;
         self.inner
-            .propose_dlc_channel(channel_details, contract_input)
+            .propose_dlc_channel(channel_details.clone(), contract_input)
             .await
             .context("Could not propose dlc channel")?;
 
+        let temporary_contract_id = self
+            .inner
+            .get_temporary_contract_id_by_sub_channel_id(channel_details.channel_id)
```

Comment:
AFAIK we have:

- An LN channel ID.
- A subchannel ID.
- A DLC channel ID.
- A contract ID.

And AFAIK `ln_channel_id == subchannel_id`.

Thus, I think it's okay to say either `_by_subchannel_id` or `_by_ln_channel_id`. I suggested the former because we are expecting it to be identifying a subchannel ID in this context, but it is just as fair to say that it should be the latter because we _know_ it is definitely a LN channel ID.